docs/csharp/language-reference/keywords/where-generic-type-constraint.md

```diff
@@ -34,7 +34,10 @@ translation.priority.ht:
 # where (generic type constraint) (C# Reference)
 In a generic type definition, the `where` clause is used to specify constraints on the types that can be used as arguments for a type parameter defined in a generic declaration. For example, you can declare a generic class, `MyGenericClass`, such that the type parameter `T` implements the <xref:System.IComparable%601> interface:  
   
-<CodeContentPlaceHolder>0</CodeContentPlaceHolder>  
+```  
```

Comment:
Added cs identifiers to this file. As you suggested, we can address changing cs identifiers to csharp in a separate PR.